concepts/asynchronous-programming/vb/readme.md

```diff
@@ -0,0 +1,36 @@
+---
+languages:
+   - vb
+ products:
+   - dotnet-core
+   - windows
+page_type: sample
+name: "Async Sample: Asynchronous Programming with Async and Await"
+urlFragment: "MainWindow.xaml.vb"
```

Comment:
```suggestion
urlFragment: "async-and-await-vb"
```
The uriFragment identifies the URL of the sample (e.g., <https://docs.microsoft.com/samples/async-and-await-vb>.